en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -4,6 +4,16 @@ Use MuleSoft Composer to build connected experiences faster. You don't need to w
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
+== July 13, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* You can now delete OAuth connectors from your flows. 
```

Comment:
1) Can remove OAuth now
2) "connectors" --> "connections"
3) I would tweak this a bit. You were able to delete connections before. The main change is that now you are alerted as to which inactive and active flows are using the connection that you would like to delete.